.asf.yaml

```diff
@@ -63,7 +63,7 @@ github:
       required_pull_request_reviews:
         dismiss_stale_reviews: false
         require_code_owner_reviews: true
-        required_approving_review_count: 1
+        required_approving_review_count: 2
```

Comment:
This is wrong.
Otherwise committers won't be able to commit their own patches with only one approval from another committer.
The rule is that you need two +1 from people with committer karma.

So if a committer sends a patch,he needs only 1 binding +1

@StevenLuMT @zymap

Reply:
Basically, I'd prefer not to set it as a strict rule since a committer should be confident to merge a typo fix. Committers share a high trust level. They know when to request one more reviewer.

Reply:
@eolivelli  @zymap  More content can be discussed, I will initiate a discussion, let’s determine the final .asf.yaml
please see the mail  [https://lists.apache.org/thread/o75xogn8r7zylqlwnno84dsm9nqoh5rs](https://lists.apache.org/thread/o75xogn8r7zylqlwnno84dsm9nqoh5rs)